src/libs/actions/IOU.ts

```diff
@@ -2117,7 +2116,7 @@ function getTrackExpenseInformation(
         iouAction,
         reportPreviewAction,
         optimisticTransactionThread ?? {},
-        (optimisticCreatedActionForTransactionThread as OptimisticCreatedReportAction) ?? {}, // Add type assertion here
+        optimisticCreatedActionForTransactionThread ?? null, // Add type assertion here
```

Comment:
```suggestion
        optimisticCreatedActionForTransactionThread ?? null,
```

Let's remove this comment